R/benchmarks.R

```diff
@@ -2,25 +2,24 @@
 #'
 #' @param run_id the run_id to get benchmarks for (default: `NULL`, list all)
 #' @param batch_id the batch_id to get benchmarks for (default: `NULL`, list all)
+#' @param run_reason a string to specify the run reason (default: `NULL`, list all)
 #' @inheritParams runs
 #'
 #' @return the response
 #' @export
-benchmarks <- function(run_id = NULL, batch_id = NULL, simplifyVector = TRUE, flatten = TRUE, ...) {
+benchmarks <- function(run_id = NULL, batch_id = NULL, run_reason = NULL, simplifyVector = TRUE, flatten = TRUE, ...) {
```

Comment:
Not something you have to change now, but on the server side we've been trying to clarify what we mean when we say "benchmark", i.e. a benchmark result or a conceptual benchmark, and renaming accordingly. So this would become `benchmark_results()` probably.

Reply:
Oh thank you! This is absolutely something I want to do here.